<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuditClientWidget</class>
 <widget class="QWidget" name="AuditClientWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>693</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AuditClientWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>461</height>
    </rect>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>621</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>                                                                                    Audit Client</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>614</width>
       <height>230</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>70</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditAdresseIP">
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Num√©ro de port du serveur :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Adresse IP du serveur :</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEditPort">
            <property name="text">
             <string>1025</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButtonQuitter">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Quitter</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonConnexion">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Connexion</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Etat de la connexion</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="textEditEtat"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxInformationsClient">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>601</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Informations Client</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>571</width>
       <height>114</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButtonOrdinateur">
          <property name="text">
           <string>Nom de l'ordinateur distant</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonUtilisateur">
          <property name="text">
           <string>Nom de l'utilisateur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonArchitecture">
          <property name="text">
           <string>Architecture processeur</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonOs">
          <property name="text">
           <string>OS de l'ordinateur</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="lineEditOrdinateur"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditUtilisateur"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditArchitecture"/>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEditOs"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonQuitter</sender>
   <signal>clicked()</signal>
   <receiver>AuditClientWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>106</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>676</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
